autoload/lsp.vim

```diff
@@ -697,3 +698,29 @@ endfunction
 function! lsp#complete(...) abort
     return call('lsp#omni#complete', a:000)
 endfunction
+
+let s:event_queue = []
+let s:event_timer = -1
+
+function! lsp#add_event_queue(queue) abort
```

Comment:
Any particular reason `lsp#add_event_queue` is public? I find this internal implementation of vim-lsp so would be better to make it private if possible.